style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: inline;
+  width: 320px;
+  align-content: column;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  align-content: flex-start;
+}
+
+.menu-ham {
+  display: block;
+  flex-shrink: 3;
+  flex-basis: flex-end;
+  order: 0;
+  margin-right: 19px;
+}
+
+.name {
+  margin-left: 16px;
+  margin-bottom: 20px;
+  margin-top: 30px;
+  font-family: 'Playfair Display', sans-serif;
+  font-size: 50px;
+  font-weight: 400px;
+  line-height: 50px;
+  color: #1a2236;
+  stroke: black 1px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
- [ ] I appreciate for going far and beyond the requirement, but there is no `text shadow` in the Figma design and you should stick to the design as part of the project requirement

Figma design
![header-text-shadow](https://user-images.githubusercontent.com/44057863/164610810-86bb0c88-31bb-4c8f-b33f-81f933a657bf.png)
.

Reply:
im sure there is a shadow. And a border on text of 1 px